### PR TITLE
more namespacing of packages

### PR DESCRIPTION
**Pull request by robertzk**
Note that `evaluate::is.error` is never actually used, so I removed it from the `#' @imports` section and the NAMESPACE file. If it *is* used metaprogramatically and I failed to grep for it as such, I will restore it.

Otherwise, this pull request follows in the spirit of https://github.com/yihui/knitr/pull/999.

### CODE REVIEW

File: R/package.R [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #'   + \code{weave}).
 #' @references Full documentation and demos: \url{http://yihui.name/knitr/};
 #'   FAQ's: \url{http://bit.ly/knitr-faq}
-#' @importFrom evaluate evaluate is.recordedplot is.source is.warning is.error
```
Comment (robertzk) [inline R/package.R]: Removed due to lack of usage. I may be wrong if metaprogramming is at play.

**Comment by yihui** [comment]
Again, I hope you can get rid of the second commit. If you want to sync up with the master branch, you need to pull _with rebase_:

    git checkout more_namespacing
    git pull --rebase https://github.com/yihui/knitr.git more_namespacing

`git rebase more_namespacing~2 -i` may help you clean up the history (use `fix` on the second commit)

File: R/package.R (hunk 1 above)
Comment (yihui) [inline R/package.R]: Seems good to me.

**Comment by robertzk** [comment]
Thank you, I know what went wrong on my end. Does it look better now?

**Pull request merged**